core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/components/edit-ema-palette/components/edit-ema-palette-content-type/edit-ema-palette-content-type.component.html

```diff
@@ -24,15 +24,15 @@
             *ngFor="let contenttype of contentTypes; let i = index"
```

Comment:
This `*ngFor` will handle a lot of items? By default I think all ngFor needs to use trackBy

Reply:
TrackBy Added!